docs/tutorials/deployments.md

```diff
@@ -456,17 +457,17 @@ INFO:     Uvicorn running on http://127.0.0.1:4200 (Press CTRL+C to quit)
     </div>
 
 
-## Agents and work queues
+## Agents and work pools
 
-As mentioned at the beginning of this tutorial, you still need two more items to run orchestrated deployments: an agent and a work queue. You'll set those up next.
+As mentioned at the beginning of this tutorial, you still need two more items to run orchestrated deployments: an agent and a work pool. You'll set those up next.
 
-[Agents and work queues](/concepts/work-queues/) are the mechanisms by which Prefect orchestrates deployment flow runs in remote execution environments.
+[Agents and work pools](/concepts/work-pools/) are the mechanisms by which Prefect orchestrates deployment flow runs in remote execution environments.
 
-Work queues let you organize flow runs into queues for execution. Agents pick up work from one or more queues and execute the runs.
+work pools let you organize flow runs into queues for execution. Agents pick up work from one or more queues and execute the runs.
 
-In the Prefect UI, you can create a work queue by selecting the **Work Queues** page, then creating a new work queue. However, in our case you don't need to manually create a work queue because it was created automatically when you created your deployment. If you hadn't created your deployment yet, it would be created when you start your agent. 
+In the Prefect UI, you can create a work pool by selecting the **work pools** page, then creating a new work pool. However, in our case you don't need to manually create a work pool because it was created automatically when you created your deployment. If you hadn't created your deployment yet, it would be created when you start your agent. 
```

Comment:
```suggestion
In the Prefect UI, you can create a work pool by selecting the **Work Pools** page, then creating a new work pool. However, in our case you don't need to manually create a work pool because it was created automatically when you created your deployment. If you hadn't created your deployment yet, it would be created when you start your agent. 
```